pyproject.toml

```
@@ -0,0 +1,10 @@
+[tool.scikit-build]
+cmake.minimum-version = "3.26.4"
```

Would recommend also specifying a couple extra things:
```suggestion
cmake.minimum-version = "3.26.4"
ninja.make-fallback = true
build-dir = "build/{wheel_tag}"
```
Specifying the build-dir is necessary for editable installs, which Graham wants to make sure are supported.